### PR TITLE
fix for redirection

### PR DESCRIPTION
**Pull request by kotsios5**
closes https://github.com/yclas/yclas/issues/2332

### CODE REVIEW

File: oc/classes/auth/oc.php [hunk 1]
```diff
@@ -429,7 +429,7 @@ public function ql_login($quicklogin=NULL)
      */
     public function login_redirect()
     {
-        HTTP::redirect(Core::request('auth_redirect',Route::url('oc-panel')));
+        HTTP::redirect(Route::url('oc-panel'));
```
Comment (neo22s) [inline oc/classes/auth/oc.php]: but now he gets wrongly redirected. This was meant to redirect the user to where he was before. If we remove it he will not be back where he was.

**Comment by neo22s** [comment]
@kotsios5 check this cant be the issue ;)

File: oc/classes/auth/oc.php (hunk 1 above)
Comment (neo22s) [inline oc/classes/auth/oc.php]: and put this back as how it was. no? since this is wrong ;)

Comment (neo22s) [inline oc/classes/auth/oc.php]: reverted ;)

Comment (kotsios5) [inline oc/classes/auth/oc.php]: ups! forgot to revert.

**Pull request merged**